Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "plane-split"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Given the `euclid` types are exposed in the public API, this needs to have a minor bump to `0.4.0`